content/en/observability_pipelines/setup/splunk.md

```diff
@@ -338,16 +338,32 @@ sources:
     address: 0.0.0.0:8088
     valid_tokens:
         - $${SPLUNK_TOKEN}
+  splunk_tcpout:
+    type: socket
+    mode: tcp
+    address: 0.0.0.0:8089
```

Comment:
Wait. The `socket` source was removed from OPW.

Reply:
Ah, that's right. I have an older image of OPW, it seems, that it hasn't been removed from. We should maybe push back at this now that we have a use case that really requires it.

Reply:
@silversupreme, I've added you as an editor to the [OPW Security Risks](https://docs.google.com/document/d/1GjAFWv2xlb-yNd58tpkKgKkKzPUw_2ZwkdkpkhFlug4/edit?usp=sharing) doc. 

Could you write up a blurb arguing for `socket`, and then tag @arbll, @barieom and OCR to open for discussion?